thunx-spring-api/src/main/java/com/contentgrid/thunx/spring/data/rest/AbacConfiguration.java

```diff
@@ -36,7 +40,9 @@ public AbacExceptionHandler exceptionHandler() {
 
     @Bean
     public AbacRequestFilter abacFilter(ThunkExpressionDecoder thunkDecoder) {
-        return new AbacRequestFilter(thunkDecoder);
+        var allowMissingAbac = applicationContext.getEnvironment()
+                .getProperty("contentgrid.thunx.allow-missing-abac", boolean.class, false);
```

Comment:
You can also inject the property directly in an argument with `@Value`: https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/value-annotations.html

That is a bit tighter-scoped than injecting the whole application context